roles/create_client_library_conf/templates/conf.json.j2

```diff
@@ -12,9 +15,22 @@
                     "port": {{ endpoint.port }}
                 }{% if not loop.last %},
 {% endif %}
+{% endfor %}
+
+            ]{% if gen2 | bool %},{% endif %}
+            {% if gen2 | bool %}
+            
+            "rdd_endpoints": [
+{% for endpoint in cluster.endpoints %}
+                {
+                    "ipv4": "{{ endpoint.ipv4 }}",
+                    "port": {{ rdd_port }}
+                }{% if not loop.last %},
+{% endif %}
 {% endfor %}
 
             ]
+            {% endif %}
```

Comment:
Need to remove the whitespace on line #33 - then the json formatting should be 👌👌